src/components/ProfilePage.tsx

```diff
@@ -0,0 +1,221 @@
+import React, { useState, useEffect } from 'react'
+import { displayToast } from 'synapse-react-client/dist/containers/ToastMessage'
+import { getFileHandleByIdURL, getMyUserBundle, updateMyUserProfile, uploadFile } from 'synapse-react-client/dist/utils/SynapseClient'
+import { FileUploadComplete, UserBundle, UserProfile } from 'synapse-react-client/dist/utils/synapseTypes'
+import { useSynapseContext } from 'synapse-react-client/dist/utils/SynapseContext'
+import { Button, Col, Container, FormControl, FormGroup, FormLabel, Row } from 'react-bootstrap'
+import MailIcon from 'assets/mail.svg'
+import LinkIcon from 'assets/link.svg'
+import EditIcon from 'assets/RedEditPencil.svg'
+import VerifiedBorder from 'assets/VerifiedProfilePic.svg'
+import { SynapseConstants, Typography } from 'synapse-react-client'
+
+export type ProfilePageProps={}
+
+export const ProfilePage = (props: ProfilePageProps) => {
+    const { accessToken } = useSynapseContext()
+
+    const [ userProfile, setUserProfile ] = useState<UserProfile>()
+    const [ firstName, setFirstName ] = useState<string|undefined>()
+    const [ lastName, setLastName ] = useState<string|undefined>()
+    const [ position, setPosition ] = useState<string|undefined>()
+    const [ company, setCompany ] = useState<string|undefined>()
+    const [ location, setLocation ] = useState<string|undefined>()
+    const [ summary, setSummary ] = useState<string|undefined>()
+    const [ url, setUrl ] = useState<string|undefined>()
+    const [ profilePicUrl, setProfilePicUrl ] = useState<string|undefined>()
+    const [ fileHandleId, setFileHandleId ] = useState<string|undefined>()
+    const [ verified, setVerfied ] = useState<boolean>()
+    const [ editing, setEditing ] = useState(false)
+
+    const getProfile = async () => {
+        try{
+            const mask =
+            SynapseConstants.USER_BUNDLE_MASK_IS_VERIFIED |
+            SynapseConstants.USER_BUNDLE_MASK_USER_PROFILE
+
+            const bundle: UserBundle = await getMyUserBundle(mask, accessToken)
+            console.log(bundle.userProfile)
+            let picUrl
+            if(bundle.userProfile?.profilePicureFileHandleId){
+                picUrl = await getFileHandleByIdURL(bundle.userProfile?.profilePicureFileHandleId as string,accessToken)
```

Comment:
Since the user will always be the one who uploads the profile picture (creator of the file handle), this should work!

Reply:
There is a dedicated service for this (in case you want to look at someone else's profile picture): http://rest-docs.synapse.org/rest/GET/userProfile/profileId/image.html